build.gradle

```diff
@@ -340,7 +344,20 @@ task macAppImage(dependsOn: 'createBundlesDir', type: Exec) {
                 '--main-jar', 'mucommander-'+project.version+'.jar', \
                 '--main-class', 'com.mucommander.main.muCommander', \
                 '--runtime-image', 'jre/macOS', \
-                '--java-options', '--add-exports java.desktop/com.apple.laf=ALL-UNNAMED --add-exports java.desktop/com.apple.eio=ALL-UNNAMED --add-exports java.desktop/com.apple.eawt=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED --add-opens java.transaction.xa/javax.transaction.xa=ALL-UNNAMED --add-opens java.management/javax.management=ALL-UNNAMED --add-opens java.rmi/java.rmi=ALL-UNNAMED --add-opens java.security.jgss/org.ietf.jgss=ALL-UNNAMED --add-opens java.sql/java.sql=ALL-UNNAMED --add-opens java.base/sun.net.www.protocol.http=ALL-UNNAMED --add-opens java.base/sun.net.www.protocol.https=ALL-UNNAMED', \
+                '--java-options', '\
+                        --add-exports java.desktop/com.apple.laf=ALL-UNNAMED \
+                        --add-exports java.desktop/com.apple.eio=ALL-UNNAMED \
+                        --add-exports java.desktop/com.apple.eawt=ALL-UNNAMED \
+                        --add-opens java.desktop/javax.swing.plaf.basic=ALL-UNNAMED \
+                        --add-opens java.base/java.io=ALL-UNNAMED \
```

Comment:
added this line (with javax.swing.plaf.basic). Should I also add it to other places? Like the one above or in mucommander.sh?

Is there some similar place for other platforms like unix/windows? @ahadas

Reply:
yeah, it should also be added to `java-options` in `winAppImage` and `linuxAppImage`
`mucommander.sh` is used in tgz and portable packages now and I don't regularly use them - seems like it missed few things.. but anyway, as for this change - better to add it there as well

Reply:
done